dm/config/checking_item.go

```diff
@@ -35,6 +35,7 @@ const (
 	ShardAutoIncrementIDChecking = "auto_increment_ID"
 	OnlineDDLChecking            = "online_ddl"
 	BinlogDBChecking             = "binlog_db"
+	ConnAmountChecking           = "conn_amount"
```

Comment:
should use conn_number. amount is uncountable. Also update other naming.

Reply:
89e2d9d